src/pharmpy/modeling/expressions.py

```diff
@@ -884,6 +884,56 @@ def get_rv_parameters(model: Model, rv: str) -> List[str]:
     return sorted(map(str, _filter_symbols(dependency_graph, free_symbols, {sympy.Symbol(rv)})))
 
 
+def get_rv_names(model: Model, parameter: str, var_type: str = 'iiv') -> List[str]:
+    """Retrieves name of iiv parameters in :class:`pharmpy.model.Model` given a parameter.
+
+    Parameters
+    ----------
+    model : Model
+        Pharmpy model to retrieve parameters from
+    parameter : str
+        Name of parameter to retrieve random variable from
+    var_type: str
+        Variability type: iiv (default) or iov
+
+    Return
+    ------
+    list[str]
+        A list of random variable names for the given parameter
+
+    Example
+    -------
+    >>> from pharmpy.modeling import *
+    >>> model = load_example_model("pheno")
+    >>> get_rv_names(model, 'CL')
+    ['ETA_1']
+
+    See also
+    --------
+    get_rv_parameters
+    has_random_effect
+    get_pk_parameters
+    get_individual_parameters
+
+    """
+    if parameter not in list(map(str, model.statements.free_symbols)):
```

Comment:
Would it make sense here to also check that it's not e.g. `ETA_1` or so that is inputted? Since etas are not parameters?